diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter4/NullChecksRule.kt

```diff
@@ -140,36 +140,45 @@ class NullChecksRule(configRules: List<RulesConfig>) : DiktatRule(
                         |${elseCodeLines.joinToString(separator = "\n")}
                         |} ?: break
                         """.trimMargin()
-                else -> """
-                        |$variableName?.let {
-                        |${elseCodeLines.joinToString(separator = "\n")}
-                        |}
-                        |?: run {
-                        |${thenCodeLines.joinToString(separator = "\n")}
-                        |}
-                    """.trimMargin()
+                else -> getDefaultCaseNullCheck(variableName, elseCodeLines, thenCodeLines)
```

Comment:
What about other cases? I believe, we can extract this logic of adjusting blocks (what you call `editedThenPart` and `editedElsePart`) and replace all `joinToString`s in other branches with it

Reply:
Done